README.md

```diff
@@ -7,57 +7,87 @@
 </p>
+<br/>
+
+The SuluRedirectBundle is compatible with Sulu **starting from version 2.0**. Have a look at the `require` section in
+the [composer.json](https://github.com/sulu/SuluRedirectBundle/blob/2.x/composer.json) to find an
+**up-to-date list of the requirements** of the bundle.
+
+## 🚀&nbsp; Installation and Documentation
+
+Execute the following [composer](https://getcomposer.org/) commands to add the bundle to the dependencies of your
+project:
+
+```bash
+composer require sulu/redirect-bundle
+```
+
+Afterwards, visit the [bundle documentation](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc/README.md) to
+find out **how to set up and configure the SuluRedirectBundle** to your specific needs.
+
+## 💡&nbsp; Features
 
-The SuluRedirectBundle adds support for managing redirects in Sulu.
+### Importing redirects
 
-Additional features included:
+One of the great features of this bundle is the ability to import redirects from a CSV file.
+The most simplified file just contains two columns, `source` and `target`.
+Of course, all the other options like `statusCode`, `sourceHost`, and `enabled` can also be set in the import file.
 
-* Content-Manager can create new redirect-routes in Sulu-Admin
-* Import redirect-routes from csv
+It’s also possible to override existing redirects with an import; you just have to set the same value for `source`.
 
-## Status
+### Enabling and disabling redirects
 
-This repository will become version 1.0 of SuluRedirectBundle. It is under **heavy development** and currently its APIs
-and code are not stable yet (pre 1.0).
+Sometimes it’s necessary to prepare redirects which are not ready yet, especially if you have a large number of redirects.
+Of course there’s a solution for that — toggler in the toolbar to enable or disable redirects quickly.
+This allows you to import a large number of disabled redirects and then check them in the administration interface,
+before enabling them when needed.
 
-## Requirements
+### Different statuses
 
-* Composer
-* PHP `^7.2`
-* Sulu `^2.0.0`
+The SuluRedirectBundle comes with three different statuses to be used for redirects:
 
-For detailed requirements see [composer.json](https://github.com/sulu/SuluRedirectBundle/blob/2.x/composer.json).
+- **301** Moved permanently
+- **302** Moved temporarily
+- **410** Gone
 
-## Documentation
+Those status codes are explained in detail [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-The the Documentation is stored in the
-[Resources/doc/](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc) folder.
+### Automatic `Gone` redirects
 
-## Installation
+This bundle also adds the possibility to automatically create redirects with status `410 Gone`, if a page or a entity with a route has been removed. This is very useful, because now search engines know, that that page has been deleted.
```

Comment:
```suggestion
This bundle also adds the possibility to automatically create redirects with status `410 Gone`, if a page or a entity with a route has been removed. This is very useful, because now search engines know, that this page has been deleted.
```